code/game/machinery/cryopod.dm

```diff
@@ -398,29 +398,8 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		if(target.mind.assigned_role in GLOB.command_positions)
 			alert("You're a Head of Staff![generic_plsnoleave_message]")
 			caught = TRUE
-		if(isovermind(target))
-			alert("You're a Blob![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_changeling(target))
-			alert("You're a Changeling![generic_plsnoleave_message]")
-			caught = TRUE
-		if(iscultist(target) || is_servant_of_ratvar(target))
-			alert("You're a Cultist![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_devil(target))
-			alert("You're a Devil![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_gangster(target) || is_gang_boss(target))
-			alert("You're a Gangster![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_nuclear_operative(target))
-			alert("You're a Nuclear Operative![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_revolutionary(target) || is_head_revolutionary(target))
-			alert("You're a Revolutionary![generic_plsnoleave_message]")
-			caught = TRUE
-		if(is_wizard(target))
-			alert("You're a Wizard![generic_plsnoleave_message]")
+		if(target.mind.has_antag_datum(/datum/antagonist))
```

Comment:
You could probably replace "antagonist" with the actual title of the antag

Reply:
Possible and a good idea, just don't know how/where an example of this is. [antag_flag]?

Reply:
I would suggest rephrasing this section to

```
var/datum/antagonist/A = target.mind.has_antag_datum(/datum/antagonist)
if(A)
    alert("You're \a [A.name]![generic_plsnoleave_message]")
```

or something along the lines of that

Reply:
Implemented.